### PR TITLE
Fixes for openfl-next

### PR DESCRIPTION
**Pull request by larsiusprime**

### CODE REVIEW

File: include.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	
 	<assets path="assets/sounds" include="*.ogg" if="html5" unless="unit-test" />
 	<assets path="assets/sounds" include="*.mp3" if="flash" embed="true" />
-	<assets path="assets/sounds" include="*.ogg" unless="flash || html5" embed="true" />
```
Comment (Gama11) [inline include.xml]: Was this causing issues / does lime legacy not support embedding .oggs?

If so, we should at least embed them when `lime_legacy` is defined.

File: flixel/system/debug/BitmapLog.hx [hunk 2]
```diff
@@ -272,7 +272,10 @@ class BitmapLog extends Window
 			_entries[i] = null;
 		}
 		_entries = [];
-		_canvas.fillRect(_canvas.rect, FlxColor.TRANSPARENT);
+		if (_canvas != null && _canvas.rect != null)
```
Comment (Gama11) [inline flixel/system/debug/BitmapLog.hx]: Under what circumstances did this cause issues?

**Comment by Gama11** [comment]
@larsiusprime Are these changes still relevant?

**Comment by Gama11** [comment]
Closed due to lack of followup.

**Comment by larsiusprime** [comment]
Hey sorry about the delay on this one; I did this to get next up and running right around when the transition started, but if you've gotten next to run without it then it shouldn't be an issue.

**Pull request closed without merging**